python/hail/representation/genomeref.py

```diff
@@ -0,0 +1,199 @@
+from hail.java import handle_py4j, jiterable_to_list
+from hail.typecheck import *
+from hail.representation.interval import Interval
+from hail.utils import wrap_to_list
+from hail.history import *
+
+
+class GenomeReference(HistoryMixin):
+    """An object that represents a genome reference.
+
+    :param str name: Name of reference
+
+    :param contigs: Contig names
+    :type contigs: list of str
+
+    :param lengths: Dict of contig names to contig lengths
+    :type lengths: dict of str to int
+
+    :param x_contigs: Contigs to be treated as X chromosomes
+    :type x_contigs: str or list of str
+
+    :param y_contigs: Contigs to be treated as Y chromosomes
+    :type y_contigs: str or list of str
+
+    :param mt_contigs: Contigs to be treated as mitochondrial chromosomes
```

Comment:
I think its more common to speak of `mitochondrial DNA`.

https://en.wikipedia.org/wiki/Mitochondrial_DNA

https://books.google.com/ngrams/graph?content=mitochondrial+DNA%2Cmitochondrial+chromosome%2Chuman+DNA%2Chuman+chromosome&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cmitochondrial%20DNA%3B%2Cc0%3B.t1%3B%2Cmitochondrial%20chromosome%3B%2Cc0%3B.t1%3B%2Chuman%20DNA%3B%2Cc0%3B.t1%3B%2Chuman%20chromosome%3B%2Cc0